src/Components/Notifications/NotificationErrorMessage.tsx

```diff
@@ -1,8 +1,18 @@
 import { Text, Box, Spacer } from "@artsy/palette"
 import { RouterLink } from "System/Router/RouterLink"
+import { useSystemContext } from "System/SystemContext"
 import { FC } from "react"
 
 export const NotificationErrorMessage: FC = () => {
+  const { isLoggedIn } = useSystemContext()
+
+  if (!isLoggedIn)
+    return (
+      <Box maxWidth={800}>
+        <Text variant="lg">Please log in to view your notifications.</Text>
+      </Box>
+    )
```

Comment:
To not show the error message `Sorry, the activity could not be loaded due to an error.` when the auth dialog is visible.